src/main/java/Todo.java

```diff
@@ -0,0 +1,16 @@
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    public void print() {
```

Comment:
The implementation of print() in Todo and Deadline is mostly similar, could possibly be moved to the base class